src/main/java/it/eng/idsa/businesslogic/configuration/IdsMessageBean.java

```diff
@@ -0,0 +1,16 @@
+package it.eng.idsa.businesslogic.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import de.fhg.aisec.ids.camel.idscp2.processors.IdsMessageTypeExtractionProcessor;
+
+@Configuration
+public class IdsMessageBean {
```

Comment:
Is this bean still relevant?
If used only to log message type in route, I think we can remove it.